runtime/bootstrap/index.ts

```diff
@@ -296,7 +296,11 @@ export async function registerBundle(
   await context.global.set('module', {}, {copy: true});
 
   // compiling the bundle allows IVM to map the stack trace.
-  const bundle = fs.readFileSync(path).toString();
+
+  // readFileSync does not properly close files in the lambda
+  const fileDescriptor = fs.openSync(path, 'r');
+  const bundle = fs.readFileSync(fileDescriptor).toString();
```

Comment:
Seems like the readFileSync implementation already calls closeSync: https://github.com/nodejs/node/blob/589b67aa1748974b24df4b4b7281e0cce4e915c7/lib/fs.js#L481

Reply:
@sam-codaio mentioned that there are reports of it not closing properly in lambda. This explicit call will close it is that right?

Reply:
If there a stack overflow or something about a lambda-specific bug the code can link to?